ios/keyman/Keyman/Keyman/Classes/InfoViewController/InfoViewController.swift

```diff
@@ -13,14 +13,21 @@ import Reachability
 class InfoViewController: UIViewController, UIWebViewDelegate {
   @IBOutlet var webView: UIWebView!
 
+  @IBOutlet var backButton: UIBarButtonItem!
+  @IBOutlet var forwardButton: UIBarButtonItem!
+
   private var networkReachable: Reachability?
 
-  convenience init() {
+  private var exitClosure: (() -> Void)? = nil;
+
+  convenience init(exitClosure: @escaping () -> Void) {
```

Comment:
Yes, it's awkward... but unfortunately, the info view is rather oddly spaghetti-coded with the app's main view.  Both the standard text area _and_ the info view are created at app init, and both are always active.  Just... only one or the other is set to visible.  As a result, the info view can't _actually_ be dismissed with a standard `.dismiss()` call.

I wonder if this implementation detail shines light on something that may have been underlying for #6631?  I noticed some similar effects to underlying behaviors with some of my attempts to work around this PR's layout issues to a few things that happened while investigating that PR.

Reply:
Honestly, this spaghetti-coded aspect is probably deserving of an issue... but unless serious value can be shown, probably not one to be immediately addressed given how close release is.